### ISSUE TITLE
Relatives place of birth should just be city, state, county

### ISSUE THREAD

**Issue by billy22g** [truetandem/e-QIP-prototype#1189]
**What did you do?**
Relatives place of birth shows full address. Should only show the city/state/country.

<img width="659" alt="screen shot 2017-05-09 at 4 16 56 pm" src="https://cloud.githubusercontent.com/assets/5091167/25875107/09eb8a94-34d3-11e7-9c85-c0996ed3663b.png">

In app currently:
<img width="687" alt="screen shot 2017-05-09 at 4 15 33 pm" src="https://cloud.githubusercontent.com/assets/5091167/25875111/0e5070ea-34d3-11e7-84b0-992734b2b45d.png">

**Comment by floresj** [comment]
@Jay-Neighbours-I72 Are we doing away with APO/FPO in this case? Do you think we should always render a city/state/country or would it be preferable to have United States/International buttons to toggle the display of the state.

**Comment by Jay-Neighbours-I72** [comment]
It will be a case by case basis - this specific example doesn't need APO/FPO

There may be other cases in the form where we do need to keep it.

**Comment by floresj** [comment]
@Jay-Neighbours-I72 Excellent. To clarify, so **always** render city/state/country?

**Comment by billy22g** [comment]
Good point John on international. I think it does need to satisfy that case @Jay-Neighbours-I72. What about if my mom was born in Zurich? We use a similar pattern in Section 1's Place of Birth, I would just mimic that but only ask for:

City/State/Country in US
City/Country outside US

Obviously this needs to be tracked as part of the content tracking for Donna and OMB.

**Comment by Jay-Neighbours-I72** [comment]
That makes sense to me, I'll put a placeholder in the content tracking doc and complete it when I can screen cap the prototype

**Comment by billy22g** [comment]
Have we figured out an elegant way to accept countries that no longer exist today? If someone was born in Zaire, they get an error now. cc: @Jay-Neighbours-I72 this feels like someone where we could use some more thought put into it from a design perspective. It's an issue when folks can enter multiple countries as well.

<img width="668" alt="screen shot 2017-05-22 at 11 04 38 am" src="https://cloud.githubusercontent.com/assets/5091167/26320085/cd910586-3ede-11e7-8a69-29ada329df6e.png">

**Comment by bmallred** [comment]
@billy22g I'll add it to #1027!
